README.md

```diff
@@ -52,6 +52,14 @@ API documentation is provided with Swagger:
 
 NOTE: Swagger scans classes in the `uk.gov.hmcts.reform.pdfservice.controllers` package.
 
+
+### Service authentication
```

Comment:
Why is this needed, the pdf-service stores no user data, it has no state and is a very simple service.
What benefit does authentication bring

Reply:
The project separates pdf service from cmc to be under platform group for any service to be able to retrieve desired pdfs. As per project plan - S2S Auth is a requirement. Hence the Jira link